po/pt_BR.po

```diff
@@ -103,11 +103,11 @@ msgstr "Título:"
 #: src/Dialogs/NewAlarmDialog.vala:148
 #, fuzzy
```

Comment:
```suggestion
```

You need to remove "fuzzy" which means "needs editing", otherwise the updated translations would never be shown.